Client/qtTeamTalk/keycomp.ui

```diff
@@ -69,7 +69,7 @@
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
-         <string>This dialog closes when you have released all keys</string>
+         <string>This dialog will close automaticly when you released all keys</string>
```

Comment:
This correction is minimal and will invalidate all existing translations. I would prefer to stay with the original